app/assets/javascripts/application.js

```diff
@@ -34,6 +34,8 @@
 //= require jquery/jquery-ui-1.9.2.custom.min
 //= require bootstrap
 //= require bootstrap-hover-dropdown/bootstrap-hover-dropdown.min
+//= require c3/c3
```

Comment:
why is this referenced with duplication c3/c3 and not just c3 for example?
I'm just wondering because the instructions on https://rails-assets.org/ show examples where the name is specified only once in require statement. thanks